.babelrc

```diff
@@ -0,0 +1,19 @@
+{
+  "presets": [
+    [
+      "@babel/preset-env",
+      {
+        "targets": {
+          "node": "current"
+        },
+        "modules": "commonjs"
+      }
+    ],
+    "@babel/preset-typescript"
+  ],
+  "plugins": [
+    [
+      "babel-plugin-add-import-extension", { "extension": "cjs" }
```

Comment:
This is still necessary because the babel CLI flag only affects output files but they are not reflected in `import/require`